api/overview/azure/batch.md

```diff
@@ -29,12 +37,44 @@ Install-Package Azure.Batch
 dotnet add package Azure.Batch
 ```
 
+### Example
+
+The following example uses the client SDK to create a job to run in Azure Batch.
+
+```csharp
+/*
+using Microsoft.Azure.Batch.Auth;
+using Microsoft.Azure.Batch.Common;
+*/
+using (BatchClient batchClient = await BatchClient.OpenAsync(credentials))

```

Comment:
Adding a line showing the creation of BatchSharedKeyCredentials, used in OpenAsync, might be helpful